samples/Test.AppWriter/Program.cs

```diff
@@ -20,99 +27,180 @@ private static IServiceProvider ConfigureServiceProvider()
         return serviceProvider;
     }
 
-    // This produces a simple example app including the specified number of screens and a few generic controls
-    // This is intended for testing purposes only
-    private static App GetExampleApp(IServiceProvider provider, string appName)
+    //private static string? ValidateFilePath()
```

Comment:
+1 to @tehcrashxor 's feedback. Commented out code should have a reason to be left here. If there is no good reason then just remove it because anyone can always go back in git history .